packages/vdaf/src/index.ts

```diff
@@ -98,12 +115,51 @@ export abstract class Vdaf<
     });
   }
 
+  encodeInputShare(inputShare: InputShare): Buffer {
+    inputShare;
+    return Buffer.alloc(0);
+  }
+
+  encodePublicShare(publicShare: PublicShare): Buffer {
+    publicShare;
+    return Buffer.alloc(0);
+  }
+
+  encodeAggregatorShare(aggregatorShare: AggregatorShare): Buffer {
+    aggregatorShare;
+    return Buffer.alloc(0);
+  }
```

Comment:
Should these be abstract instead? (same for the test vector methods returning empty strings or buffers below)

Reply:
I wasn't sure if these should be required, which they would be if they were abstract. This was related to my confusion about serialization, vdaf, prio3, and dap.

Reply:
made them abstract (and required) and added a few other encoding signatures to vdaf